lib/NodeStuffPlugin.js

```diff
@@ -69,6 +69,13 @@ class NodeStuffPlugin {
 									);
 								}
 							});
+						parser.hooks.canRename.for("global").tap(
```

Comment:
there is no need to prevent renaming for `__dirname`/`__filename` since they should be always strings